### ISSUE TITLE
4C5387D6 - Superstars V8 Racing Demo

### ISSUE THREAD

**Issue by voltagex** [xenia-project/game-compatibility#53]
Tested with benvanik/xenia@fc7695f
Boots via default.xex, makes it to legal/disclaimer screen and then xenia crashes.
![untitled](https://cloud.githubusercontent.com/assets/159567/8374128/2e3ed930-1c36-11e5-9dcf-c7c1b76c9971.png)
http://marketplace.xbox.com/en-US/Product/Superstars-V8-NC/66acd000-77fe-1000-9115-d8024c5307d8 (possibly wrong)

**Comment by Razzile** [comment]
Have you tried running it through visual studio? I noticed doing that seems to "fix" a lot of the assertion failures and games seem more stable. Whether there is an actual reason for that I have no idea.

**Comment by voltagex** [comment]
No, I don't understand the build process well enough yet - these binaries were created with `xb build`

Hoping @benvanik will chime in on my reports and tell me how to get more debug output.

**Comment by voltagex** [comment]
Way too many asserts to get through it by hitting ignore.

**Comment by voltagex** [comment]
benvanik/xenia@47a3ec0

Xenia's image loading has definitely changed - none of my old UDF disc images will load.
FMVs / intros seem to be corrupted.

![image](https://user-images.githubusercontent.com/159567/27507345-81c2832c-5910-11e7-845f-dd08d751423b.png)

Menu isn't quite right either
![image](https://user-images.githubusercontent.com/159567/27507350-94ea5cc2-5910-11e7-8422-0f8ad0273720.png)

Goes ingame but isn't rendering right

![image](https://user-images.githubusercontent.com/159567/27507355-b526f8ba-5910-11e7-992d-5998c0f2ebd3.png)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/1099553/xenia.zip)
